cudax/cmake/cudaxSTFConfigureTarget.cmake

```
@@ -11,10 +11,16 @@ function(cudax_stf_configure_target target_name)
     CUDA::curand
     CUDA::cuda_driver
   )
+
+  if (cudax_ENABLE_CUDASTF_CODE_GENERATION)
+    target_compile_options(${target_name} PRIVATE
+      $<$<COMPILE_LANG_AND_ID:CUDA,NVIDIA>:--extended-lambda>
+    )
```

Should this also add
```suggestion
    target_compile_options(${target_name} PRIVATE
      $<$<COMPILE_LANG_AND_ID:CUDA,NVIDIA>:--extended-lambda>
    )
    target_compile_definitions(${target_name} PRIVATE "CUDASTF_CODE_GENERATION">)
```